packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/modules/Module.kt

```diff
@@ -1,13 +1,15 @@
 package expo.modules.kotlin.modules
 
+import android.content.Context
 import android.os.Bundle
 import expo.modules.core.errors.ModuleDestroyedException
 import expo.modules.kotlin.AppContext
 import expo.modules.kotlin.providers.AppContextProvider
+import expo.modules.kotlin.providers.ContextProvider
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.cancel
 
-abstract class Module : AppContextProvider {
+abstract class Module : AppContextProvider, ContextProvider {
```

Comment:
at the first shot to this line, i was getting confused. do you think whether it's a good idea for merging `ContextProvider -> AppContextProvider`?